internal/bminventory/inventory.go

```diff
@@ -2304,6 +2326,20 @@ func (b *bareMetalInventory) getHost(ctx context.Context, clusterId string, host
 	return &host, nil
 }
 
+func (b *bareMetalInventory) getCluster(ctx context.Context, clusterID string) (*common.Cluster, error) {
```

Comment:
you are using this function as `IsClusterExists` so why no create it like this?

Reply:
wanted to have one that will return me cluster object for future usage